docs/docs/cmd/spo/group/group-user-add.md

```diff
@@ -0,0 +1,36 @@
+# spo group user add
+
+Add a user or multiple users to SharePoint Group
+
+## Usage
+
+```sh
+m365 spo group user add [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the SharePoint group is available
+
+`-i, --groupId <groupId>`
+: Id of the SharePoint Group to which user needs to be added
+
+`-n, --userName <userName>`
```

Comment:
I didn't catch it in the spec, but let's remove the short `-n` here. It would be ambiguous if we ever extended the command with support for specifying `groupName`.

Reply:
Sure. Have updated the [specs](https://github.com/pnp/cli-microsoft365/issues/2483#issue-916289528)